### PR TITLE
[Enhancement] Add more bvars

### PR DESCRIPTION
**Pull request by sduzh**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: be/src/service/service_be/lake_service.cpp [hunk 1]
```diff
@@ -35,9 +36,64 @@
...
+        return nullptr;
+    }
+    auto as = env->agent_server();
+    if (UNLIKELY(as == nullptr)) {
+        return nullptr;
+    }
+    return as->get_thread_pool(type);
+}
+
+static int get_queued_tasks(TTaskType::type type) {
```
Comment (kevincai) [inline be/src/service/service_be/lake_service.cpp:55]:
```suggestion
static int get_num_of_queued_tasks(TTaskType::type type) {
```

File: be/src/service/service_be/lake_service.cpp [hunk 2]
```diff
@@ -35,9 +36,64 @@
...
+    auto tp = get_thread_pool(type);
+    if (UNLIKELY(tp == nullptr)) {
+        return 0;
+    }
+    return tp->num_queued_tasks();
+}
+
+static int get_active_tasks(TTaskType::type type) {
```
Comment (kevincai) [inline be/src/service/service_be/lake_service.cpp:63]: is this consistent that "active tasks == active_thread"?

**Review by kevincai — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by sduzh** [comment]
@Mergify rebase

**Review by wyb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by Seaven — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**